packages/invoices/src/OutboundShipment/ListView/ListView.tsx

```diff
@@ -101,7 +101,7 @@ export const OutboundShipmentListViewComponent: FC = () => {
     onChangePage,
     pagination,
     invalidate,
-  } = useListData({ key: 'color' }, 'invoice', OutboundShipmentListViewApi);
+  } = useListData({ key: 'TYPE' }, 'invoice', OutboundShipmentListViewApi);
```

Comment:
the sorting is only on a small subset of columns and they have to be shouty case. Thought to just to do this for now.